src/browserCommands.ts

```diff
@@ -0,0 +1,880 @@
+/**
+ * The command IDs used by the file browser plugin.
```

Comment:
This is a complete copy of upstream, with very small changes, right?
So please add that information in the comment at the top. If there are many changes, please add comments on the specific places detailing what's different (to make it easier to update in the future)

Reply:
It's not a full copy, I have copied what we needed and do some small changes, mostly exports so it could be in a separate file, also changed the paths for file creation. I added a comments

Reply:
It's still not clear to me what the small changes are. Please annotate them.
Also provide the link to the file (in the specific commit) you took this code.